src/classes/RD2_ElevateIntegrationService_TEST.cls

```diff
@@ -654,8 +654,8 @@ public class RD2_ElevateIntegrationService_TEST {
             .withStatusClosed()
             .build();
         npe03__Recurring_Donation__c oldRd = newRd.clone();
-        newRd.LastElevateEventPlayed__c = 'event-v1';
-        oldRd.LastElevateEventPlayed__c = 'event-v1';
+        newRd.LastElevateVersionPlayed__c = 1;
```

Comment:
Given your code, I know it will pass, but as defense against future code modifications, it might be good to write a unit test for the case where the previous version is null.

Reply:
Great find, I pushed an update to test these scenarios.